public/v2/apps/nextcloud.json

```diff
@@ -81,5 +82,10 @@
         "label": "admin password",
         "description": "Password for the Nextcloud admin user.",
         "validRegex": "/.{1,}/"
+    },{
+        "id": "$$cap_http_https_cors",
+        "label": "will you force https ?",
+        "description": "cors configuration to login are set by the docker image, if you want to use https, please set as https, if not just set as http",
```

Comment:
A few things here:
1- Change the label to "Protocol of proxy"
2- Add this to the description: `Choose either http or https`
3- Let's add a default value to make it easier for the end user. Perhaps `http`.

Reply:
Done.

I also add a little informations about nginx optimisation for configuration compliance.
I'll let you checking that :)